cmd/rosa/main.go

```diff
@@ -105,3 +113,27 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
+	rprtr := reporter.CreateReporterOrExit()
+
+	currVersion, err := gversion.NewVersion(info.Version)
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem verifying the current version of rosa. Please be aware that you are possibly running an outdated version: %w", err)
+		os.Exit(1)
+	}
+
+	latestVersionFromMirror, err := rosa.RetrieveLatestVersionFromMirror()
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem retrieving latest version from mirror. Please be aware that you are possibly running an outdated version: %w", err)
+		os.Exit(1)
```

Comment:
Similar to the above comment - would we rather either of the following?

a.) Prompt to continue and skip the version comparison
b.) Continue without any further prompts.